e2e/testdata/fn-eval/missing-fn-config/.expected/config.yaml

```diff
@@ -15,3 +15,4 @@
 testType: eval
 exitCode: 1
 image: gcr.io/kpt-fn/set-namespace:v0.1
+stdErr: "failed to configure function: input namespace cannot be emptyerror: exit status 1"
```

Comment:
line break issue.

As part of adding tests, can you also audit the error messages for content and style? That's the spirit of this issue.

go/kpt-errors-style-guide

Reply:
Will do

Reply:
Could you clarify what you mean by line break issue? I don't see anything about line breaks in the errors style guide

Reply:
what's `emptyerror`?

Reply:
Taking a step back, if we put our end-users' hats on: does this sentence make sense?

Reply:
Ah, understood. Thank you.

Reply:
Not resolved?

Reply:
Resolved, thanks. Filed the issue here https://github.com/GoogleContainerTools/kpt/issues/1793